### PR TITLE
Adds import map file, and more VS Code workspace configurations

### PR DESCRIPTION
**Pull request by rodbecc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,18 +16,18 @@ Add the **Deno** binary to the PATH variable in your shell configuration file.
...
+echo 'export DENO_INSTALL="/Users/$USER/.deno"' >> ~/.zshrc
 echo 'export PATH="$DENO_INSTALL/bin:$PATH"' >> ~/.zshrc
 ```
 
 Install **Velociraptor** to watch files, list available scripts, and run
 predefined scripts:
 
 ```sh
-deno install -qA -n vr https://deno.land/x/velociraptor/cli.ts
+deno install -qA -n vr https://deno.land/x/velociraptor@1.1.0/cli.ts
```
Comment (vvasc) [inline README.md:27]: is this working?

---
**Sema Reaction:** :question: I have a question

File: velociraptor.ts [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
+
 export default <ScriptsConfiguration> {
   scripts: {
     install_docker_aliases: {
       cmd: "deno run scripts/install_aliases.ts docker",
       desc: "Adds to your shell config file Docker and Docker Compose aliases",
-      allow: ["env", "read", "write"],
-      unstable: true,
-      watch: false,
+      ...common,
```
Comment (vvasc) [inline velociraptor.ts:15]: nice

File: velociraptor.ts [hunk 3]
```diff
@@ -1,27 +1,28 @@
...
+  allow: ["env", "read", "write"],
+  unstable: true,
+  watch: false,
+  imap: "./import_map.json",
+};
+
 export default <ScriptsConfiguration> {
   scripts: {
     install_docker_aliases: {
       cmd: "deno run scripts/install_aliases.ts docker",
```
Comment (vvasc) [inline velociraptor.ts:13]: what do you think about use `deno run` as a variable?

---
**Sema Reaction:** :question: I have a question

File: README.md [hunk 4]
```diff
@@ -16,18 +16,18 @@ Add the **Deno** binary to the PATH variable in your shell configuration file.
 For ZSH shell, for instance:
 
 ```sh
-echo 'export DENO_INSTALL="/Users/yourUser/.deno"' >> ~/.zshrc
+echo 'export DENO_INSTALL="/Users/$USER/.deno"' >> ~/.zshrc
```
Comment (vvasc) [inline README.md:19]: nice

**Review by vvasc — APPROVED** [review]
---
**Sema Reaction:** :ok_hand: This code looks good

File: README.md (hunk 1 above)
Comment (vvasc) [reply README.md:27]: I checked, so it is

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Comment (rodbecc) [reply README.md:27]: Yep! I tested it before commiting.

File: velociraptor.ts (hunk 3 above)
Comment (rodbecc) [reply velociraptor.ts:13]: Do you mean to change it as a shell alias? Can you elaborate?

**Pull request merged**